lib/mix/lib/mix/tasks/profile.eprof.ex

```diff
@@ -26,7 +26,7 @@ defmodule Mix.Tasks.Profile.Eprof do
     * `--matching` - only profile calls matching the given `Module.function/arity` pattern
     * `--calls` - filters out any results with a call count lower than this
     * `--time` - filters out any results that took lower than specified (in µs)
-    * `--sort` - sort the results by `time` or `calls` (default: `time`)
+    * `--sort` - sorts the results by `time` or `calls` (default: `time`)
     * `--config`, `-c` - loads the given configuration file
     * `--eval`, `-e` - evaluate the given code
```

Comment:
Should the description of `--eval` and `--no-warmup` use “evaluates” and “skips”  respectively?